spk/owfs/src/dsm-control.sh

```diff
@@ -0,0 +1,97 @@
+#!/bin/sh
+
+# Package
+PACKAGE="owfs"
+DNAME="Owfs"
+
+# Others
+INSTALL_DIR="/usr/local/${PACKAGE}"
+PATH="${INSTALL_DIR}/bin:${PATH}"
+
+OW_SERVER="${INSTALL_DIR}/bin/owserver"
+SERV_PID_FILE="${INSTALL_DIR}/var/owserver.pid"
+
+OW_HTTPD="${INSTALL_DIR}/bin/owhttpd"
+HTTPD_PID_FILE="${INSTALL_DIR}/var/owhttpd.pid"
+
+start_daemon ()
+{
+    ${OW_SERVER} -c ${INSTALL_DIR}/var/owfs.conf --pid-file ${SERV_PID_FILE}
+    sleep 1
+	${OW_HTTPD} -c ${INSTALL_DIR}/var/owfs.conf --pid-file ${HTTPD_PID_FILE}
+    sleep 1
+}
+
+stop_daemon ()
+{
+    kill `cat ${HTTPD_PID_FILE}`
+    wait_for_status 1 20
+    rm -f ${HTTPD_PID_FILE}
+	kill `cat ${SERV_PID_FILE}`
+    wait_for_status 1 20
+    rm -f ${SERV_PID_FILE}
+}
+
+daemon_status ()
+{
+    if [ -f ${HTTPD_PID_FILE} ] && [ -d /proc/`cat ${HTTPD_PID_FILE}` ]; then
```

Comment:
Fix indents throughout this function.
Along with that, why not use the same function we use almost everywhere else:
` if [ -f ${HTTPD_PID_FILE} ] && kill -0 `cat ${HTTPD_PID_FILE}` > /dev/null 2>&1; then`